Runtime/Tracking/Velocity/ArtificialVelocityApplier.cs

```diff
@@ -1,189 +1,55 @@
 ﻿namespace Zinnia.Tracking.Velocity
```

Comment:
Do the tests still pass for these updates?

Reply:
![image](https://user-images.githubusercontent.com/22952143/100373421-b99ec980-3045-11eb-8d25-363a5cb6547b.png)
yes they are.